apps/docs/components/reference/RefSectionHandler.tsx

```diff
@@ -60,6 +60,10 @@ const RefSectionHandler = (props: RefSectionHandlerProps) => {
   const pageTitle = getPageTitle()
   const section = props.sections.find((section) => section.slug === slug)
   const fullTitle = `${pageTitle}${section ? ` - ${section.title}` : ''}`
+  const path = router.asPath
```

Comment:
Did you specifically decide against just using a regular replace?

Reply:
You're right - will have better performance. I'll update.